interfaces/Portalicious/src/app/components/registrations-table/registrations-table.component.ts

```diff
@@ -114,6 +108,13 @@ export class RegistrationsTableComponent {
       getCellChipData: (registration) =>
         getChipDataByRegistrationStatus(registration.status),
     },
+    {
+      field: 'registrationCreated',
+      fieldForFilter: 'registrationCreatedDate',
+      header: $localize`:@@registration-created:Registration created`,
```

Comment:
I wonder if its possible to get some 'system' in these names...
And how to check/enforce that system... 🤔 

In the old-portal there where also a few concepts used side-by-side.... so I don't know which should be applied everywhere.. ;)

But we should be able to provide some more context to translators with these.. And also to get some more "consistent" translations easily etc.

Reply:
By "system" you mean some kind of convention / naming guideline / taxonomy?

Indeed that would be interesting but I don't have an opinion / recommendation as of now. And also out of scope for this PR (I'm just moving code around here).